vm/src/vector_3.cpp

```diff
@@ -0,0 +1,63 @@
+#include "vector_3.h"
+#include "deserializer.h"
+#include "matrix_4.h"
+#include "serializer.h"
+#include <cassert>
+
+namespace Borges {
+
+Vector_3::Vector_3(std::initializer_list<double> v)
+{
+    assert(v.size() == 3);
+    for (std::size_t i = 0; i < 3; ++i) {
+        v_[i] = v.begin()[i];
```

Comment:
why not use v.at(i) here ?

Reply:
This is an std::initializer_list, not a std::vector. std::initializer_list::at() does not exist.

Reply:
duh